.sbtopts

```diff
@@ -0,0 +1,5 @@
+-J-Xms512M
+-J-Xmx1536M
+-J-Xss4M
```

Comment:
this changed from 1M, assuming on purpose?

Reply:
yes - I think one of the upgrades is very recursive at compilation, so I was getting stack overflows at compile time; this increases the stack so compilation can succeed